workflow_examples/readme.md

```diff
@@ -0,0 +1,82 @@
+# Workflow API Usage
```

Comment:
I suggest to move general documentation about how to use the api (link to openapi spec, short example how to use), to the main README.MD.